requirements-dev.txt

```diff
@@ -1,3 +1,4 @@
+-r requirements.txt
```

Comment:
why ?

Usually the way to go is:
* `pip install -r requirements.txt`
* `pip install -r requirements-dev.txt`
* `pip install -e .[test]`

It makes the setup more complicated.

Reply:
Is it? then we should implement it. ATM in branch master the workflow of `make pyenvisntall` is slightly different:

1. `pyenv` --> `pip install  -U pip wheel setuptools; pip install -r requirements.txt`
2. `pyenvinstall` --> `pip  install -e .\[test\] ; sha256sum requirements*.txt > ./local/py3/requirements.sha256`

ATM in branch master the `requirements-dev.txt` is never used by a pip command.  But `pip  install -e .\[test\]` will read from this file

But I will try to implement what you suggested.

Reply:
'pyenv' now installs 'requirements*.txt' (see commit message 8d8d96)